docs/create-packages/Creating-a-Package.md

```diff
@@ -3,20 +3,26 @@ title: How to create a NuGet package
 description: A detailed guide to the process of designing and creating a NuGet package, including key decision points like files and versioning.
 author: karann-msft
 ms.author: karann
-ms.date: 12/12/2017
+ms.date: 05/24/2019
 ms.topic: conceptual
 ---
 
 # Creating NuGet packages
 
-No matter what your package does or what code it contains, you use `nuget.exe` to package that functionality into a component that can be shared with and used by any number of other developers. To install `nuget.exe`, see [Install NuGet CLI](../install-nuget-client-tools.md#nugetexe-cli). Note that Visual Studio does not automatically include `nuget.exe`.
+No matter what your package does or what code it contains, you use one of the NuGet CLI tools, either `nuget.exe` or `dotnet.exe`, to package that functionality into a component that can be shared with and used by any number of other developers. To install NuGet CLI tools, see [Install NuGet client tools](../install-nuget-client-tools.md). Note that Visual Studio does not automatically include a CLI tool.
+
+- For .NET Core and .NET Standard projects that use the SDK-style format ([SDK attribute](/dotnet/core/tools/csproj#additions)), NuGet uses information in the project file directly to create a package. For details, see [Create .NET Standard Packages with Visual Studio 2017](../quickstart/create-and-publish-a-package-using-visual-studio.md) and [NuGet pack and restore as MSBuild targets](../reference/msbuild-targets.md).
+
+- For non-SDK-style projects, follow the steps described in this article to create a package.
```

Comment:
I once asked around internally what we should call non-SDK style projects and "traditional project" was suggested. I like "non-SDK-style project" better, but do we have a standard name for it? NuGet's docs shouldn't be using a different name if dotnet or msbuild docs already have a different name for it.

Reply:
We have the the "SDK-style project" term published in .NET:
https://docs.microsoft.com/en-us/dotnet/standard/library-guidance/nuget
https://docs.microsoft.com/en-us/dotnet/core/tools/dotnet-msbuild
So, it should be OK for now. The other term I've seen is "project SDK".